src/features/metadata-based-view/__tests__/IconWithTooltip.test.js

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+import { IconWithTooltipComponent as IconWithTooltip } from '../IconWithTooltip';
+
+import messages from '../../../elements/common/messages';
+import { CANCEL_ICON_TYPE, EDIT_ICON_TYPE, SAVE_ICON_TYPE } from '../constants';
+
+describe('features/metadata-based-view/IconWithTooltip', () => {
+    const getWrapper = props => mount(<IconWithTooltip {...props} />);
+    const onClick = () => {};
+
+    test('Get IconClose with Tooltip to cancel edited value', () => {
```

Comment:
Minor: the tests in this repo seem to follow a naming convention of "(lowercase) should _verb_..."